covid_model/input/attribute_multipliers.json

```diff
@@ -30,5 +30,54 @@
 
   {"param": "severe_immunity", "attrs": {"immun": ["strong", "weak"]}, "val": 0.975, "desc": "Vaccinated immunity vs severe disease is very high (among vacc. successes)."},
   {"param": "severe_immunity", "attrs": {"immun": ["strong", "weak"], "vacc": "none"}, "val": 0.5, "desc": "Prior infection immunity vs severe disease is much weaker."},
-  {"param": "severe_immunity", "attrs": {"variant": ["delta", "omicron"]}, "mult": 0.9, "desc": "Delta and Omicron reduce immunity to severe disease."}
+  {"param": "severe_immunity", "attrs": {"variant": ["delta", "omicron"]}, "mult": 0.9, "desc": "Delta and Omicron reduce immunity to severe disease."},
```

Comment:
redundant but very explicit adjustments to the initial seeds for each variant in each region. Adjusted levels are just scaled by the population relative to statewide population.